tests/integration/ambient/cacert_rotation_test.go

```diff
@@ -64,13 +64,13 @@ func TestIntermediateCertificateRefresh(t *testing.T) {
 
 			// Update CA with new intermediate cert
 			if err := cert.CreateCustomCASecret(t,
-				"ca-cert-alt-2.pem", "ca-key-alt-2.pem",
-				"cert-chain-alt-2.pem", "root-cert-alt.pem"); err != nil {
+				"ca-cert.pem", "ca-key.pem",
+				"cert-chain.pem", "root-cert.pem"); err != nil {
```

Comment:
This is broken since the test asserts we are using the same root (its testing intermediates). But I don't understand how we generated these (broken) certs to regenerate them correctly. There seems to be no code for this.

@zirain maybe you can help?

Reply:
https://github.com/istio/istio/blob/1ee1a434356b1ba94c3e763886dc03b064ebd925/samples/certs/README.md#L16

I don't know why @GregHanson didn't use `ca-cert.pem` and `ca-cert-alt.pem` in the beginning.

Reply:
Sorry, haven't been following this debug too closely.  @zirain here is the discussion for why I had to generate new ones:
https://github.com/istio/istio/pull/48168#discussion_r1423096841

Reply:
@howardjohn cert should have been generated with the script `security/samples/plugin_ca_certs/gen_certs.sh` or using the steps [here](https://istio.io/latest/docs/tasks/security/cert-management/plugin-ca-cert/#plug-in-certificates-and-key-into-the-cluster)

This should be the same script @hzxuzhonghu used when writing the root cert rotation test

Reply:
hmm, ran this script and updated a bit to try to automate things but failing some tests still.